.travis.yml

```diff
@@ -2,14 +2,18 @@ language: python
 python:
     - 2.6
     - 2.7
+    - 3.3
+    - 3.4
 before_install:
     - sudo add-apt-repository "deb http://us.archive.ubuntu.com/ubuntu/ trusty main restricted universe multiverse"
     - sudo add-apt-repository "deb http://us.archive.ubuntu.com/ubuntu/ trusty-updates main restricted universe multiverse"
     - sudo apt-get update -qq
     - sudo apt-get install libev-dev/trusty
 install:
-    - if [[ $TRAVIS_PYTHON_VERSION == '2.6' ]]; then pip install unittest2; fi
+    - if [[ $TRAVIS_PYTHON_VERSION == '2.6' ]]; then pip install unittest2 ordereddict; fi
```

Comment:
Add `OrderedDict` support for Python 2.6 to make the one test fully compatible with all 5 Python flavors supported.